src/styles/global.css

```
@@ -1,1081 +1,216 @@
-    --color-muted: var(--text-muted);
-    --color-inverse: var(--text-inverse);
-    --color-border-subtle: var(--border-subtle);
-    --color-border-default: var(--border-default);
-    --color-border-strong: var(--border-strong);
-    --color-accent-ui: var(--color-accent);
-    --color-accent-muted-ui: var(--color-accent-muted);
+@import 'tailwindcss';
+
+/* SiteHeader toggles these scroll-state classes imperatively, so they aren't
+   discoverable in markup — safelist them explicitly. */
+@source inline('border-line/70 bg-bg/65 backdrop-blur-md py-3 md:py-3.5');
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)